src/interfaces/ccip/IEVM2EVMOnRamp.sol

```diff
@@ -3,8 +3,9 @@
 pragma solidity ^0.8.0;
 
 import {IInternal} from './IInternal.sol';
+import {ITypeAndVersion} from './ITypeAndVersion.sol';
 
-interface IEVM2EVMOnRamp {
+interface IEVM2EVMOnRamp is ITypeAndVersion {
```

Comment:
it does not change between versions?

Reply:
nope, this interface has been kept minimal (only methods used in the AIPs so far were added to this interface)